oeps/processes/oep-0057-core-repositories.rst

```diff
@@ -0,0 +1,166 @@
+
+However, from these many repositories, we wish to enumerate a smaller set which:
+
+* is a good target for enhancement, documentation, and technical investment;
+
+* defines a logical boundary within which it is fair to enforce Open edX Proposals;
+
+* defines an administrative boundary within which the Core Contributor Program applies;
+
+* makes the Open edX project more comprehensible by collecting its most critical components;
+
+* (TODO: should I add something about tCRIL stewardship/ownership?); and
+
+* encompasses the first-party aspect of the Open edX release supply chain, identifying repositories that need to be:
+
+  * monitoried for discloures under the `Open edX security policy`_,
+  * regularly updated with security patches,
+  * covered by automated unit tests and static analysis, and
+  * ready to deploy to production at any time.
+
+
+Considerations
+--------------
+
+
+* Resolving legacy processes and access policies
+
+  * The process for reviewing addition/removal of repos via the Core Contributor program isn't ready to be spun up yet or even well-defined. In the meantime, tCRIL has been making these judgement calls.
+  * 2U OCM and some of tCRIL retain historical access to many Core repositories outside of the Core Contributor program process. There is a plan to bring those folks under the Core Contributor umbrella, but it will take time.
+  * tCRIL is currently using a few public repositories in the openedx organization for day-to-day work, such as ``openedx/tcril-engineering``. Upon acceptance of this OEP, they would need to make a new GitHub organization for those repositories.
+
+* Cleaning up the Core
+
+  * Many repositories in the openedx GitHub organization do not currently meet the Core criteria. We will need to review the organization contents, and for several repos, transfer them out or invest in them.
+  * Even so, the definition of the Core still captures a large number of repositories, probably larger than we want our "Core" to be. We could reduce it by loosening the package dependencies of many services to being optional instead of strictly required.
+
+* Ensuring the Core contains everything it should
+
+  * There are four repositories directly included in Maple that aren't in the openedx organization. We'll either need to remove them from the release or transfer them into openedx.
+  * This will take a while, but we'll need to look at the list of packages that are depended upon by released repositories. If any of them are Open edX code but outside the openedx organization, we'll need to move them into openedx, or refactor the dependencies so that they're optional.
+
+
+
+Motivation
+----------
+
+TODO
+
+
+Alternatives Considered
```

Comment:
The big picture of this OEP as I understand it is:
- Define criteria for a set of repos to be called Core, and
- Make that set be the same as the openedx GitHub set.

I'm wondering if it's a good idea to join those two ideas together.  As [Dave commented](https://github.com/openedx/open-edx-proposals/pull/312/files#r819158716), orgs imply access administrations, and as [Sarina pointed out](https://github.com/openedx/open-edx-proposals/pull/312/files#r816013821), there will be data to migrate if some repos have to be moved.

Couldn't we find other ways of clearly indicating what is Core?  As written, this OEP will require making yet another organization (or maybe more than one?) The phrase "open edx" refers to a software platform, but also a project.  Why shouldn't the openedx organization include repos that are in the project but not the platform?

Reply:
It might make the proposal more concrete if we could see a dry-run of what would have to move.  What repos need to move into the openedx org, what repos would need to move out, and where they would go.